fractions/javaee/ejb-remote/pom.xml

```diff
@@ -44,6 +44,11 @@
       <groupId>org.wildfly.swarm</groupId>
       <artifactId>ejb</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>messaging</artifactId>
+      <optional>true</optional>
```

Comment:
Is this OK? I couldn't find a fraction that would _optionally_ depend on another fraction... Or maybe would `<scope>provided</scope>` be better?

Reply:
I think this is better, as it won't make `messaging` content available on classpath to anything that depends just on `ejb-remote`. But I think provided would